test/data/data.js

```diff
@@ -88,6 +97,30 @@ function populate(db) {
     }),
   );
 
+  prom.push(
+    insertDocument(db, {
+      $type: 'token',
+      $kind: 'entry',
+      $owner: 'b@b.com',
+      $id: 'myReadOnlyToken',
+      $creationDate: 0,
+      uuid: 'A',
+      rights: ['read'],
+    }),
+  );
+
+  prom.push(
+    insertDocument(db, {
+      $type: 'token',
+      $kind: 'entry',
+      $owner: 'b@b.com',
+      $id: 'myAddAttachmentToken',
+      $creationDate: 0,
+      uuid: 'A',
+      rights: ['read', 'addAttachment'],
```

Comment:
```suggestion
      rights: ['read', 'write', 'addAttachment'],
```

The "write" right is necessary to upload an attachment